roles/openshift_logging_elasticsearch/tasks/generate_secret.yaml

```diff
@@ -9,6 +9,11 @@
   register: passwd_file
   check_mode: no
 
+- when: passwd_file.stat.exists
+  slurp:
+    src: "{{ generated_certs_dir }}/passwd.yml"
+  register: _logging_metrics_proxy_passwd
+
```

Comment:
we can probably remove this block since we do this same thing after we create the file if it doesn't exist